school/teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './school/person'
+
+class Teacher < Person
+  def initialize(specialization)
```

Comment:
- [ ] The same thing goes here with a plus the speciaization.

Reply:
Alright, thank you.